VideoLocker/src/main/java/org/edx/mobile/view/NavigationFragment.java

```diff
@@ -234,7 +234,11 @@ public void onClick(View view) {
             public void onClick(View v) {
                 String to = environment.getConfig().getFeedbackEmailAddress();
                 String subject = getString(R.string.email_subject);
-                String email = "";
+
+                String versionName = PropertyUtil.getManifestVersionName(getActivity());
+                String osVersionText = String.format("%s %s", getString(R.string.android_os_version), android.os.Build.VERSION.RELEASE);
+                String appVersionText = String.format("%s %s", getString(R.string.app_version), versionName);
+                String email = osVersionText + "\n" + appVersionText;
```

Comment:
If you fix it, while you're there, could you rename this function? It doesn't send email It just opens the email client...